README.md

```diff
@@ -11,11 +11,11 @@ Usage: canonical_data_syncer [options]
 
 Options:
   -e, --exercise <slug>        Only sync this exercise
-  -c, --check                  Check if there missing tests. Doesn't update the tests. Terminates with a non-zero exit code if any one or more test cases are missing.
-  -d, --default <mode>         What to do with missing test cases. Allowed values: c[hoose], i[nclude], e[xclude]
+  -c, --check                  Check if there are missing tests. Doesn't update the tests. Terminates with a non-zero exit code if one or more tests are missing
+  -m, --mode <mode>            What to do with missing test cases. Allowed values: c[hoose], i[nclude], e[xclude]
```

Comment:
I've gone back and forth on this. Maybe we should add something that explains that setting this means things are handled automatically, whereas not setting it requires interaction.